crates/dekaf/src/log_appender.rs

```diff
@@ -494,17 +494,17 @@ impl<W: TaskWriter + 'static> TaskForwarder<W> {
     pub fn send_stats(&self, collection_name: String, stats: ops::stats::Binding) {
         if stats
             .left
-            .is_some_and(|s| s.bytes_total == 0 || s.docs_total == 0)
```

Comment:
I realized that this was too restrictive and was complaining too much -- it's fine to emit stats where both `bytes_total` and `docs_total` are 0 because of the filter [here](https://github.com/estuary/flow/blob/29e8bc6f744ff28e7e2322cb8c03c4ab58ae5dbc/crates/dekaf/src/log_appender.rs#L56-L60)

Reply:
Why is this filtering required in the first place? How does it come to be that bytes_total != 0 but docs_total == 0, or vice versa?

Can you instead make this an assert()-ion rather than a tracing::error?